internal/routes/all_routes.go

```diff
@@ -4,47 +4,60 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/RHEnVision/provisioning-backend/internal/config"
+	"github.com/RHEnVision/provisioning-backend/api"
+	"github.com/RHEnVision/provisioning-backend/internal/routes/helpers"
 	s "github.com/RHEnVision/provisioning-backend/internal/services"
 	"github.com/go-chi/chi/v5"
+	redoc "github.com/go-openapi/runtime/middleware"
 	"github.com/redhatinsights/platform-go-middlewares/identity"
 )
 
-func SetupRoutes(r *chi.Mux) {
-	r.Get("/ping", s.StatusService)
-	r.Mount(pathPrefix(), apiRouter())
+func redocMiddleware(handler http.Handler) http.Handler {
+	opt := redoc.RedocOpts{
+		SpecURL: fmt.Sprintf("%s/openapi.json", helpers.PathPrefix()),
+	}
+	return redoc.Redoc(opt, handler)
 }
 
-func pathPrefix() string {
-	return fmt.Sprintf("/api/%s", config.Application.Name)
+func SetupRoutes(r *chi.Mux) {
+	r.Get("/ping", s.StatusService)
+	r.Route("/docs", func(r chi.Router) {
+		r.Use(redocMiddleware)
+		r.Get("/openapi.json", api.ServeOpenAPISpec)
+	})
+	r.Mount(helpers.PathPrefix(), apiRouter())
 }
 
 func apiRouter() http.Handler {
 	r := chi.NewRouter()
-	r.Use(identity.EnforceIdentity)
-	r.Route("/sources", func(r chi.Router) {
-		r.Get("/", s.ListSources)
-		r.Route("/{ID}", func(r chi.Router) {
-			r.Get("/", s.GetSource)
+
+	r.Get("/openapi.json", api.ServeOpenAPISpec)
+	r.Group(func(r chi.Router) {
```

Comment:
I created this group for excluding the `openapi` endpoint  from the identity middleware